09-big-three/SOLUTION.md

```diff
@@ -84,6 +84,35 @@ We could optimize the placement logic by utilizing a binary search sort of techn
 
 With this, we have a linear solution and we only had to make one pass.
 
+## Solution 3: The triple variable
```

Comment:
@mwhitman189 could you sync your fork? https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

I moved this already to the other section